shotover-proxy/examples/redis-cluster-ports-rewrite/docker-compose.yml

```diff
@@ -1,90 +1,9 @@
 version: '2'
 services:
-  redis-node-0:
-    image: docker.io/bitnami/redis-cluster:6.0-debian-10
-    ports:
-      - "2220:6379"
-    volumes:
-      - redis-cluster_data-0:/bitnami/redis/data
-    environment:
-      - 'ALLOW_EMPTY_PASSWORD=yes'
-      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
-
-  redis-node-1:
-    image: docker.io/bitnami/redis-cluster:6.0-debian-10
-    ports:
-      - "2221:6379"
-    volumes:
-      - redis-cluster_data-1:/bitnami/redis/data
+  redis:
+    image: grokzen/redis-cluster:latest
```

Comment:
I would want to hear some justification for this change

Reply:
I wasn't able to get the other docker image running on localhost, if I queried `CLUSTER SLOTS` it would return an internal IP like `172.0.10:${PORT_NUMBER}` whereas this image returns `localhost:${PORT_NUMBER}`. This means it will work properly with the port rewrite transform and let me send all traffic through Shotover.

Reply:
Does that maybe demonstrate an issue with our port rewrite implementation?
Do we need to be rewriting ip address as well?

Reply:
Not necessarily, Shotover could be running on the same machine as each Redis node in a cluster and would be using the same IP. In that case it just needs to rewrite the port. 

We could rewrite the IP as well to make it suit more use cases though.